From 0e87c2fd0dc7a29b626723d38e9c77eeda6ec047 Mon Sep 17 00:00:00 2001
From: Zhantao Tang <zhantao.tang@windriver.com>
Date: Fri, 13 May 2022 14:41:44 +0800
Subject: [PATCH] dts: fsl-s32g: pfe: modify BMU2 reserved memory node to fix
 pfe probe failure

The commit a85753a61ddb ("dts: fsl-s32g: pfe: Place BMU2 inside an
internal SRAM memory region") updates BMU2 reserved memory node, but
it will cause the pfe probe fail due to:
        BMU2 buffer pool reserved mem region size exceeded

so modify the node to use privous settings to fix the issue.
And it will be updated in future.

Upstream-Status: Pending

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-s32g.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-s32g.dtsi b/arch/arm64/boot/dts/freescale/fsl-s32g.dtsi
index b69cccc0bae9..c5b88dd69a4d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-s32g.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-s32g.dtsi
@@ -35,10 +35,10 @@ reserved-memory {
 		#size-cells = <2>;
 		ranges;
 
-		pfe_reserved_bmu2: pfebufs@34000000 {
+		pfe_reserved_bmu2: pfebufs@83000000 {
 			compatible = "fsl,pfe-bmu2-pool";
-			/* BMU2: 512 KB */
-			reg = <0 0x34000000 0 0x80000>;
+			/* 2 MB */
+			reg = <0 0x83000000 0 0x200000>;
 			no-map;
 			status = "okay";
 		};
-- 
2.25.1

